zephyrdoom/src/doom/deh_misc.c

```diff
@@ -15,157 +15,168 @@
 // Parses "Misc" sections in dehacked files
 //
 
+#include "deh_misc.h"
+
 #include <stdlib.h>
 #include <string.h>
 #include <strings.h>
 
-#include "doomtype.h"
 #include "deh_defs.h"
 #include "deh_io.h"
 #include "deh_main.h"
-#include "deh_misc.h"
+#include "doomtype.h"
 
-// Dehacked: "Initial Health" 
+// Dehacked: "Initial Health"
 // This is the initial health a player has when starting anew.
 // See G_PlayerReborn in g_game.c
 
+#undef deh_initial_health
```

Comment:
Please get rid of of these undefs. Instead, uncomment the externs and comment out the #defines in the .h file